File: python/string_comparison/src/application.py
class Application:
    def __init__(self, str_1, str_2):
        self.str_1 = str_1
        self.str_2 = str_2

    def exactly_equal_size_and_included(self):
        return self.__sort_string__(self.str_1) == self.__sort_string__(self.str_2)

    # private

    def __sort_string__(self, str):
        return ''.join(sorted(str))

File: javascript/string_comparison/src/application.js
"use strict;"

const Application = class {
  constructor(str_1, str_2) {
    this.str_1 = str_1;
    this.str_2 = str_2;
  }

  exactly_equal_size_and_included() {
    return this.__sort_string__(this.str_1) === this.__sort_string__(this.str_2);
  }

  // private

  __sort_string__(str) {
    return str.split("").sort().join("");
  }
}

export default Application
